docs/source/index.rst

```diff
@@ -17,6 +17,69 @@ Welcome to the MAAP User Documentation!
   userdata.rst
   faqs.rst
   
+<<<<<<< HEAD
```

Comment:
@spa0002 I think this needs to be removed, looks like a merge conflict

Reply:
Removed line 20